### ISSUE TITLE
osrf/gazebo:gzweb8 broken

### ISSUE THREAD

**Issue by lshamis** [osrf/docker_images#97]
$ docker run osrf/gazebo:gzweb8
    ./root/gzweb/start_gzweb.sh: not found

**Comment by ruffsl** [comment]
I think the WORKDIR that `gzweb` dockerfile inherits from `gzserver` just plan `/`.  
Could you rebuild with this change to the command default in the dockerfile:

``` diff
- CMD ./root/gzweb/start_gzweb.sh && gzserver
+ CMD ./gzweb/start_gzweb.sh && gzserver
```
Or just run:
```
docker run -it -rm osrf/gazebo:gzweb8 ./gzweb/start_gzweb.sh && gzserver
```

**Comment by lshamis** [comment]
That didn't work for me.

The working directory is `/`:

    $ docker run -it --rm osrf/gazebo:gzweb8 pwd
    /

I'm not sure why `root/gzweb` doesn't contain `start_gzweb.sh`:

    $ docker run -it --rm osrf/gazebo:gzweb8 ls ./root/gzweb
    CMakeLists.txt		 build		   deploy.sh		http	       tools
    Gruntfile.js		 cmake		   get_local_models.py	karma.conf.js  webify_models.py
    README.md		 coarse_meshes.sh  gz3d			node_modules   webify_models_v2.py
    bitbucket-pipelines.yml  codecov.yml	   gzbridge		package.json

**Comment by iche033** [comment]
The `default` branch of gzweb is actively in development. The command has changed, this [closed PR](https://github.com/osrf/docker_images/pull/87/files#diff-1a72c08c60abd54a4107129c3c6cc8e9R46) shows the command that replaces `start_gzweb.sh`

**Comment by ruffsl** [comment]
Please review https://github.com/osrf/docker_images/pull/98 to make sure the default command includes the changes you need.

**Comment by ruffsl** [comment]
I believe this has been resolved with #98 merged.
